lib/lex.h

```diff
@@ -1,13 +1,14 @@
 #ifndef _RE2C_LIB_LEX_
 #define _RE2C_LIB_LEX_
 
+#include "lib/parse.h"
 #include "src/msg/location.h"
 #include "src/regexp/re.h"
 #include "src/parse/ast.h"
 
 namespace re2c {
 
-int lex(const uint8_t*& pattern, Ast& ast);
+int lex(YYSTYPE * yylval, const uint8_t*& pattern, Ast& ast);
```

Comment:
Please remove space between type and `*`, as in `YYSTYPE*`. Here and everywhere in the patch.

Reply:
Sounds good. Dropped leading whitespace. PTAL.